src/main/resources/xml/schema/2020-1/nordic-html5.rng

```diff
@@ -4376,6 +4376,24 @@
     </define>
 
     <define name="classes">
+        <ref name="classes.without.images" />
+        <zeroOrMore>
+            <choice>
+                <value>image</value>
+                <value>image-series</value>
+            </choice>
+        </zeroOrMore>
+        <ref name="classes.without.images" />
+        <zeroOrMore>
+            <choice>
+                <value>image</value>
+                <value>image-series</value>
+            </choice>
+        </zeroOrMore>
+        <ref name="classes.without.images" />
+    </define>
+
+    <define name="classes.without.images">
```

Comment:
This still allows `image` and `image-series` because all classes are allowed through `<data type="string"/>` at the end of the definition.